README.md

```diff
@@ -24,139 +24,213 @@ See [license](LICENSE) for more information.
 
 ## Changelog
 
+#### 2021-02-18
+
+**v1.1.0**
+
+- **Variable name changes (breaking changes :bangbang:)**
+
+  - `extractionVolMl`: Change from `sampleSizeL`. Description of the variable was also changed.
+  - `sampleTypeDefault`: Change from `SampleTypeDefault`. Now consistent use of lowercase first letter.
+  - `sampleTypeOtherDefault`: Change from `SampleTypeOtherDefault`.
+  - `sampleCollectionDefault`: Change from `SampleCollectionDefault`.
+  - `sampleCollectOtherDefault`: Change from `SampleCollectOtherDefault`.
+  - `sampleStorageTempCDefault`: Change from `SampleStorageTempCDefault`.
+  - `measureFractionAnalyzedDefault`: Change from `MeasureFractionAnalyzedDefault`.
+
+
+- **New variables**
+
+- Reporter
+  - `organization`: Organization of reporter. Issue [#97](https://github.com/Big-Life-Lab/covid-19-wastewater/issues/97)
+
+- Sample
+  - `reporterID`: Reporter ID. Currently, reporterID is `WWmeasure` table but reporter for samples can be different. Issue [#93](https://github.com/Big-Life-Lab/covid-19-wastewater/issues/93)
+  - `index`: Index number in case the sample was taken multiple times. Issue [#103](https://github.com/Big-Life-Lab/covid-19-wastewater/issues/103) 
+  - `InstrumentID`: Links with the Instrument table to describe the instrument used for sampling. Issue [#104](https://github.com/Big-Life-Lab/covid-19-wastewater/issues/104) 
+
+- SiteMeasure
+  - `SampleID`: Makes sure that samples can easily be linked back to the site measurements, without the need for comparing dates. In case that multiple samples need to be linked to the same site measurement, create a comma separated list of sample IDs. 
+  
+- **Deleted variable categories (breaking changes :bangbang:)**
+  - `SiteMeasure` table, `aggregation` variable, all of the options that included normalization
+    - `sdNr`,"Standard deviation, normalized",L'�cart type normalis�
+    - `geoMnNr`,"Geometric mean, normalized",Moyenne g�om�trique normalis�e.
+    - `meanNr`,"Arithmetic mean, normalized",Moyenne arithm�tique normalis�e.
+
+- **New variable categories**
+
+  - Categories added to allow variant reporting. See `WWMeasure` table, `type` variable:
+
+    - `varB117`: Variant B.1.1.7
+    - `varB1351`: Variant B.1.351
+    - `varP1`: Variant P.1
+
+  - Updated description of `WWMeasure` table, `unit` measure. These descriptions now reference gene or variant copies.
+  - New `detected`: Gene copies or variant detected in the sampleGene or variant copies. Detected = 1. Gene or variant copiesNot detected = 0.
+  - New `porpVar`: Proportion of variant in sample.
+  - `SiteMeasure` table, `type` variable
+    - `wwBOD5c`, 5 day biochemical oxygen demand
+    - `wwPtot`, Total phosphates
```

Comment:
MECP lab typically measures total phosphorous (TP) `wwTP`; but this is fine also but not too common from my experience.

Reply:
to discuss